lib/puppet/pops/issues.rb

```diff
@@ -506,6 +506,10 @@ def self.hard_issue(issue_code, *args, &block)
     "The value '#{value}' cannot be converted to Numeric."
   end
 
+  NUMERIC_COERCION = issue :NUMERIC_COERCION, :before, :after do
+    "The string '#{before}' was automatically coerced to the numerical value #{after}"
```

Comment:
Hmm... So in master, all of these errors have been externalized. How do we want to handle making sure that new errors also get externalized? Is that something that should happen post-mergeup? And if so, how are we going to remember to do it?